api/config/settings.py

```diff
@@ -312,9 +312,9 @@
 MOBILE_ACTIVATION_TOKEN_HASH_ALGORITHM = 'sha256'
 
 # Test Settings
-TEST_PAYLOAD_PATH = str(API_DIR) + '/utils/test/'
-TEST_DATA_PATH = TEST_PAYLOAD_PATH + 'data/'
-TEST_RUNNER = 'utils.test.test_runner.CMTestRunner'
+#TEST_PAYLOAD_PATH = str(API_DIR) + '/utils/test/'
```

Comment:
Avoid committing code that is commented. If the code is not needed, delete it entirely.

Reply:
Done